Directory.Build.props

```diff
@@ -15,6 +15,8 @@
     <LangVersion>Latest</LangVersion>
     <!-- Generate snupkg package -->
     <IncludeSymbols>true</IncludeSymbols>
+    <NoWarn>1701;1702</NoWarn>
```

Comment:
Can you add a comment to explain what these two are for?

Reply:
I'll have to search again - but it was something very obscure that affected old compiler versions only or smth.

Reply:
I will honestly prefer not to add a NoWarn (especially globally) if we don't fully understand why are we getting it. If we do end up needing it, I would much rather have a pragma disable in the code that is emiting the warning so that we only scope the no warn to the case we understand on why is it happening.